clients/java/src/test/java/io/camunda/client/process/rest/ModifyProcessInstanceRestTest.java

```diff
@@ -108,66 +111,6 @@ public void shouldActivateMultipleElementsWithVariables() {
     assertVariableInstruction(variableInstructionB, EMPTY_ELEMENT_ID);
   }
 
-  @Test
-  public void shouldTerminateSingleElement() {
-    // when
-    client.newModifyProcessInstanceCommand(PI_KEY).terminateElement(ELEMENT_KEY_A).send().join();
-
-    // then
-    final ProcessInstanceModificationInstruction request =
-        gatewayService.getLastRequest(ProcessInstanceModificationInstruction.class);
-    assertRequest(request, 0, 1);
-    final ProcessInstanceModificationTerminateInstruction terminateInstruction =
-        request.getTerminateInstructions().get(0);
-    assertTerminateInstruction(terminateInstruction, ELEMENT_KEY_A);
-  }
-
-  @Test
-  public void shouldTerminateMultipleElements() {
-    // when
-    client
-        .newModifyProcessInstanceCommand(PI_KEY)
-        .terminateElement(ELEMENT_KEY_A)
-        .and()
-        .terminateElement(ELEMENT_KEY_B)
-        .send()
-        .join();
-
-    // then
-    final ProcessInstanceModificationInstruction request =
-        gatewayService.getLastRequest(ProcessInstanceModificationInstruction.class);
-    assertRequest(request, 0, 2);
-    final ProcessInstanceModificationTerminateInstruction terminateInstructionA =
-        request.getTerminateInstructions().get(0);
-    assertTerminateInstruction(terminateInstructionA, ELEMENT_KEY_A);
-    final ProcessInstanceModificationTerminateInstruction terminateInstructionB =
-        request.getTerminateInstructions().get(1);
-    assertTerminateInstruction(terminateInstructionB, ELEMENT_KEY_B);
-  }
-
-  @Test
-  public void shouldActivateAndTerminateElement() {
-    // when
-    client
-        .newModifyProcessInstanceCommand(PI_KEY)
-        .activateElement(ELEMENT_ID_A)
-        .and()
-        .terminateElement(ELEMENT_KEY_A)
-        .send()
-        .join();
-
-    // then
-    final ProcessInstanceModificationInstruction request =
-        gatewayService.getLastRequest(ProcessInstanceModificationInstruction.class);
-    assertRequest(request, 1, 1);
-    final ProcessInstanceModificationActivateInstruction activateInstruction =
-        request.getActivateInstructions().get(0);
-    assertActivateInstruction(activateInstruction, ELEMENT_ID_A, EMPTY_KEY, 0);
-    final ProcessInstanceModificationTerminateInstruction terminateInstruction =
-        request.getTerminateInstructions().get(0);
-    assertTerminateInstruction(terminateInstruction, ELEMENT_KEY_A);
-  }
-
```

Comment:
ℹ️ I simply moved those further down to better group the tests by activate, terminate, move, and mixed commands.